cmd/operator-sdk/internal/genutil/openapi.go

```diff
@@ -95,25 +93,36 @@ func OpenAPIGen() error {
 	return nil
 }
 
-func buildOpenAPIGenBinary(binDir, codegenSrcDir string) error {
-	genDirs := []string{"./cmd/openapi-gen"}
-	return buildCodegenBinaries(genDirs, binDir, codegenSrcDir)
-}
-
-func openAPIGen(binDir, hf string, fqApis []string) (err error) {
-	cgPath := filepath.Join(binDir, "openapi-gen")
-	for _, fqApi := range fqApis {
-		args := []string{
-			"--input-dirs", fqApi,
-			"--output-package", fqApi,
-			"--output-file-base", "zz_generated.openapi",
-			// openapi-gen requires a boilerplate file. Either use header or an
-			// empty file if header is empty.
-			"--go-header-file", hf,
+func openAPIGen(hf string, fqApis []string) error {
+	wd, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+	flag.Set("logtostderr", "true")
+	for _, api := range fqApis {
+		// Use relative API path so the generator writes to the correct path.
+		apiPath := "./" + api[strings.Index(api, scaffold.ApisDir):]
```

Comment:
Will `"./"` break Windows users? Can we just omit `"./"`?

Same question for `deepcopyGen` in `k8s.go`.

Reply:
It will. I'll test these changes without a relative path prefix. If the generators don't work with that removed I'll make the prefix OS-agnostic.

Reply:
The generator (more specifically, `go/build`) treats paths without a `/` (or `\` on windows) prefix as a go package, which causes an error because that package does not exist.